go.mod

```diff
@@ -108,6 +108,7 @@ require (
 	github.com/akavel/rsrc v0.8.0 // indirect
 	github.com/alecthomas/participle/v2 v2.1.1 // indirect
 	github.com/armon/go-radix v1.0.0 // indirect
+	github.com/aws/aws-sdk-go v1.50.26 // indirect
```

Comment:
why are we adding indirect dependencies manually?

The agent itself does not depend on `github.com/aws/aws-sdk-go` or `github.com/jmespath/go-jmespath`.

If we want to update indirect dependencies, we need to upgrade direct packages these dependencies are coming through.